ML_workflow_try.ipynb

```diff
@@ -0,0 +1,1450 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "#get the list of selected variables\n",
+    "selected_vars = forward_regression(X, y, 0.05, verbose =True)\n",
+    "#get the least of the least important variables that were not selected\n",
+    "not_slected_vars =[var for var in X.columns if var not in selected_vars]\n",
+    "#drop the leasr important variables from the data set\n",
+    "X_new = X.drop(not_slected_vars, axis = 1)\n",
+    "#resample the data with the selected variables\n",
+    "x_resampled, y_resampled = SMOTE_resampling(X_new, y)\n",
+    "#train and predict with the resampled selected features.\n",
+    "xtest, ytest, y_pred, probs = train_predict(x_resampled, y_resampled, diabetes_classifier)\n",
+    "\n",
+    "# plot precision recall curve\n",
+    "precision, recall, thresholds = precision_recall_curve(ytest, prob_class_1)\n",
+    "plt.xlabel('Recall')\n",
+    "plt.ylabel('Precision')\n",
+    "plt.title(\"precision recall Curve,\")  \n",
+    "plt.plot(recall,precision)\n",
+    "plt.show()\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 29,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# use the misclass rate feature to analysis this model\n",
+    "def misclass_rate_feature(\n",
```

Comment:
Sure. Since I was editing the misclass_rate function while implementing it on this dataset, I left the function body there. Now I have deleted the body.